operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/LocallyRunOperatorExtension.java

```diff
@@ -182,7 +182,7 @@ public static void applyCrd(String resourceTypeName, KubernetesClient client) {
       var crdString = new String(is.readAllBytes(), StandardCharsets.UTF_8);
       LOGGER.debug("Applying CRD: {}", crdString);
       final var crd = client.load(new ByteArrayInputStream(crdString.getBytes()));
-      crd.createOrReplace();
```

Comment:
I have documented the related content in the issue. Is there any specific reason we need to use the CSA method here?

Reply:
No, It think this is fine.